marbot-aws-account-connection.yml

```diff
@@ -0,0 +1,144 @@
+---
+# Copyright widdix GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'marbot.io: AWS account connection (https://github.com/marbot-io/monitoring-jump-start)'
+Metadata:
+  'AWS::CloudFormation::Interface':
+    ParameterGroups:
+    - Label:
+        default: 'marbot endpoint'
+      Parameters:
+      - EndpointId
+      - Stage
+    - Label:
+        default: 'Features'
+      Parameters:
+      - CloudWatchMetricGraph
+Parameters:
+  EndpointId:
+    Description: 'Your marbot endpoint ID (to get this value: select a channel where marbot belongs to and send a message like this: "@marbot show me my endpoint id").'
+    Type: String
+  Stage:
+    Description: 'marbot stage (never change this!).'
+    Type: String
+    Default: v1
+    AllowedValues: [v1, dev]
+  CloudWatchMetricGraph:
+    Description: 'Enrich CloudWatch Alarms with a metric graph.'
+    Type: String
+    Default: true
+    AllowedValues: [true, false]
+Conditions:
+  CloudWatchMetricGraphEnabled: !Equals [!Ref CloudWatchMetricGraph, 'true']
+Resources:
+  Topic:
```

Comment:
Shall we switch from SNS to EventBridge for future implementations? Was thinking about that because of our bad experience with the SNS bug some time ago.

See https://github.com/marbot-io/monitoring-jump-start/blob/05f6c954b489f4355d26bb9acc793535626287a1/marbot-elastic-beanstalk.yml#L106 for an example.